app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -194,3 +194,9 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$homescreen-content-bg-color:                   white
+$homescreen-block-bg-color:                     white
+$homescreen-footer-bg-color:                    #F8F8F8
+$homescreen-block-border-color:                 #EDEDED
```

Comment:
We are not using this color anywhere else...

E.g. the boxes at `http://localhost:5000/projects/:id` have a border color of `#EAEAEA` (probably `$content-default-border-color`).

Here I would advise to use `$homescreen-block-border-color = $content-default-border-color`